scripts/import/import_oldhgmd.pl

```diff
@@ -0,0 +1,306 @@
+    
+    $columns[0] = $max_var_feat++;
+    $columns[5] = $max_var++;
+    print $new_var_feat join ("\t", @columns), "\n";
+  }
+
+  close $old_var;
+  close $new_var_file;
+  
+  $max_var = $var->[0]->[0];
+  $max_var = $max_var + 1;
+  open(my $old_var_set, '<', "$TMP_DIR/$sorted_old_var_set") or die "Cannot open file: $!";
+  open(my $new_var_set, '>', "$TMP_DIR/$new_var_set") or die "Cannot open file: $!";
+  
+  while ( my $vset = <$old_var_set> ) {
+    chomp $vset;
+    my @columns = split("\t", $vset);
+
+    $columns[0] = $max_var++;
+    print $new_var_set join("\t", @columns), "\n";
+  }
+
+  close $old_var_set;
+  close $new_var_set;
+
+  $select_max_var->finish();
+  $select_max_feat->finish();
+}
+
+sub insert_into_variation_table  {
+  my $dbhvar = shift;
+  my $load_file = shift;
+
+  my $sql = $dbhvar->prepare(qq[INSERT INTO variation (variation_id, source_id, name, class_attrib_id, somatic, evidence_attribs, display) VALUES (?,?,?,?,?,?,?)]);
+  
+  local *FH;
+  open FH, "<", "$load_file" or die "Can not open $load_file $!";
+  while (<FH>) {
+    chomp;
+    my $variation_id = (split)[0];
+    my $source_id = (split)[1];
+    my $name = (split)[2];
+    my $class_attrib_id = (split)[3];
+    my $somatic = (split)[4];
+    my $evidence_attribs = (split)[5];
+    my $display = (split)[6];
+
+    $sql->execute($variation_id, $source_id, $name, $class_attrib_id, $somatic, $evidence_attribs, $display);
+  }
+
+  close FH;
+  $sql->finish();
+
+}
+
+sub insert_variation_feature {
+  my $dbhvar = shift;
+  my $load_file = shift; 
+
+  my $sql = $dbhvar->prepare(qq[INSERT INTO variation_feature (variation_feature_id, seq_region_id, seq_region_start, seq_region_end, seq_region_strand, variation_id, allele_string, ancestral_allele, variation_name, map_weight, source_id, consequence_types, variation_set_id, class_attrib_id, evidence_attribs) VALUES (?,?,?,?,?,?,?,?,?,?,?,?,?,?,?)]);
+
+  local *FH;
+  open FH, "<", "$load_file" or die "Can not open $load_file $!";
+  while (<FH>) {
+    chomp;
+    my $var_feat_id = (split)[0];
+    my $seq_region_id = (split)[1];
+    my $seq_region_start = (split)[2];
+    my $seq_region_end = (split)[3];
+    my $seq_region_strand = (split)[4];
+    my $variation_id = (split)[5];
+    my $allele_string = (split)[6];
+    my $ancestral_allele = (split)[7];
+    my $variation_name = (split)[8];
+    my $map_weight = (split)[9];
+    my $source_id = (split)[10];
+    my $consequence_types = (split)[11];
+    my $variation_set_id = (split)[12];
+    my $class_attrib_id = (split)[13];
+    my $evidence_attribs = (split)[14];
+
+    $sql->execute($var_feat_id, $seq_region_id, $seq_region_start, $seq_region_end, $seq_region_strand, $variation_id, $allele_string, $ancestral_allele, $variation_name, $map_weight, $source_id, $consequence_types, $variation_set_id, $class_attrib_id, $evidence_attribs);
+
+  }
+
+  close FH;
+  $sql->finish();
+}
+
+sub load_all_variation_sets {
```

Comment:
To load the variants into variation_set_variation you don't need a separate file. You can use the new variation file - you just need to new variation_id then the set_id is always the same.